src/AmbientSounds.Uwp/Views/ScreensaverPage.xaml

```diff
@@ -55,9 +56,28 @@
             Visibility="{x:Bind ViewModel.VideoPlayerVisible, Mode=OneWay}" />
 
         <computeSharp:AnimatedComputeShaderPanel
+            RenderingFailed="AnimatedComputeShaderPanel_RenderingFailed"
             ShaderRunner="{x:Bind converters:PixelShaderToShaderRunnerConverter.Convert(ViewModel.AnimatedBackgroundName), Mode=OneWay}"
             Visibility="{x:Bind ViewModel.AnimatedBackgroundVisible, Mode=OneWay}" />
 
         <controls:Screensaver x:Name="ScreensaverControl" x:Load="{x:Bind ViewModel.SlideshowVisible, Mode=OneWay}" />
+
+        <!--  Rendering error bar  -->
+        <muxc:InfoBar
+            x:Name="RenderingErrorInfoBar"
+            x:Uid="RenderingErrorInfo"
+            Title="Rendering error"
+            VerticalAlignment="Bottom"
+            x:Load="False"
+            IsOpen="True"
+            Message="There was an error that caused the rendering to fail, try restarting the app. If the issue persists, please report it by sending us an email."
```

Comment:
wait, can we remove this `Message`? this uses x:uid so the string is localized that way right?

Reply:
Yup, done in d2b6fbeee1fda34dc8ae31e67a0e550b17317ab1 👍